.env.vagrant

```diff
@@ -29,9 +29,14 @@ CDN_HOST=chs-dev.internal:8080
 
 APPEALS_API_URL=http://api.chs-dev.internal
 
+OATH_SCOPE_PREFIX=https://api.companieshouse.gov.uk/company/
 OAUTH2_TOKEN_URI=http://web.chs-dev.internal:4000/oauth2/token
 OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
+OAUTH2_REQUEST_KEY=
```

Comment:
Are OAUTH2_REQUEST_KEY, OATH_SCOPE_PREFIX and ACCOUNT_URL still needed here? I'd suggest to remove these unless needed otherwise they just cause confusion that you experienced last time.